cmake/compiler/gcc/target.cmake

```diff
@@ -80,12 +84,16 @@ if(NOT no_libgcc)
 
   assert_exists(LIBGCC_FILE_NAME)
 
-  get_filename_component(LIBGCC_DIR ${LIBGCC_FILE_NAME} DIRECTORY)
+  if (CONFIG_SPARC)
+    LIST(APPEND TOOLCHAIN_LIBS ${LIBGCC_FILE_NAME})
```

Comment:
Either add an explanation to why this needs to be different with SPARC as a comment in the code, or preferably, align with the other architecture's and avoid this patch.

Reply:
We'll postpone this change after sparc is merged.  Because, for now, we are targeting GCC in SDK, we don't need this.   I'll later open another issue just for this.